src/SDL_net.c

```diff
@@ -1335,11 +1342,11 @@ NET_DatagramSocket *NET_CreateDatagramSocket(NET_Address *addr, Uint16 port)
     return sock;
 }
 
-static int SendOneDatagram(NET_DatagramSocket *sock, NET_Address *addr, Uint16 port, const void *buf, int buflen)
+static NET_Status SendOneDatagram(NET_DatagramSocket *sock, NET_Address *addr, Uint16 port, const void *buf, int buflen)
 {
     struct addrinfo *addrwithport = MakeAddrInfoWithPort(addr, SOCK_DGRAM, port);
     if (!addrwithport) {
-        return -1;
+        return NET_FAILURE;
```

Comment:
I missed this last time, but below this, you are returning 0 instead of NET_WOULDBLOCK.

Reply:
@mkunkacnc You're referring to `return WouldBlock(err) ? 0`? That's used in two other locations, in functions that wouldn't apply to this PR. To maintain some consistency, I'm not sure that should be changed there (although it wouldn't hurt and it would have the same effect).

Actually, I will make that change in this function.

It might be that there needs to be a discussion on whether all the zeroes in the code that indicate NET_WOULDBLOCK should be changed at some point.

@icculus Where do you stand on this?